test/OptionSettlement.t.sol

```diff
@@ -1517,61 +1557,77 @@ contract OptionSettlementTest is Test, NFTreceiver {
 
     function _assertClaimAmountExercised(uint256 claimId, uint112 amount) internal {
         IOptionSettlementEngine.Underlying memory underlying = engine.underlying(claimId);
-        IOptionSettlementEngine.Option memory option = engine.getOptionFromEncodedId(claimId);
+        IOptionSettlementEngine.Option memory option = engine.getOptionForTokenId(claimId);
         uint112 amountExercised = uint112(uint256(underlying.exercisePosition) / option.exerciseAmount);
         assertEq(amount, amountExercised);
     }
 
     function _writeAndExerciseNewOption(
         address underlyingAsset,
-        uint40 exerciseTimestamp,
-        uint40 expiryTimestamp,
-        address exerciseAsset,
         uint96 underlyingAmount,
+        address exerciseAsset,
         uint96 exerciseAmount,
+        uint40 earliestExerciseTimestamp,
+        uint40 expiryTimestamp,
         address writer,
         address exerciser
     ) internal returns (uint256 optionId, uint256 claimId) {
-        (optionId,) = _createNewOptionType(
-            underlyingAsset, exerciseTimestamp, expiryTimestamp, exerciseAsset, underlyingAmount, exerciseAmount
-        );
+        (optionId,) = _createNewOptionType({
+            underlyingAsset: underlyingAsset,
+            underlyingAmount: underlyingAmount,
+            exerciseAsset: exerciseAsset,
+            exerciseAmount: exerciseAmount,
+            earliestExerciseTimestamp: earliestExerciseTimestamp,
+            expiryTimestamp: expiryTimestamp
+        });
         claimId = _writeAndExerciseOption(optionId, writer, exerciser);
     }
 
+    // TODO ensure this function is written as desired
```

Comment:
what does this TODO entail?

Reply:
Just wanted someone to check w/ fresh set of eyes. BC I'm confused about the usage of the new test helpers.